### PR TITLE
Issue 1 browse card sets

### PR DESCRIPTION
**Pull request by tam-n**
image
image
image
image

### CODE REVIEW

**Comment by tam-n** [comment]
https://github.com/tam-n/ajax-project/assets/131913803/d9629ecc-3f21-4cc4-94dc-11ab8b021661

File: index.html [hunk 1]
```diff
@@ -4,16 +4,108 @@
...
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
+
   <header>
+    <div class="row navigation-bar">
+      <div class="header" >
+        <img class="header-icon" src="images/Poké_Ball_icon.png"/>
```
Comment (shawnkost) [inline index.html]: Needs an `alt` attribute

File: index.html [hunk 2]
```diff
@@ -4,16 +4,108 @@
...
+  <main class="container">
+
+    <div class="view-set hidden">
+      <div class="row half-column display-card-column">
+        <div class="tabs-wrapper">
+          <div class="tab">View Set</div>
+          <div class="tab">My Deck</div>
+        </div>
+        <div class="display-card-wrapper">
+          <img class="display-collection-card" src="images/pokemon-card-backside.png">
```
Comment (shawnkost) [inline index.html]: Needs an `alt` attribute

File: index.html [hunk 3]
```diff
@@ -4,16 +4,108 @@
...
+        </div>
+      </div>
+      <div class="row half-column card-collection-column">
+        <div class="card-collection-wrapper">
+          <div class="set-logo-wrapper">
+            <img class="set-logo" src="">
```
Comment (shawnkost) [inline index.html]: Needs an `alt` attribute

File: index.html [hunk 4]
```diff
@@ -4,16 +4,108 @@
...
+          </div>
+        </div>
+      </div>
+      <div class="row center half-column expansion-column">
+        <div class="sub-header">
+          Pokémon TCG Expansion Series
+        </div>
+        <div class="expansion-sets column-full">
+          <div class="expansion">
+            <img class="series" src="images/scarlet-violet-logo.png">
```
Comment (shawnkost) [inline index.html]: Needs an `alt` attribute!

File: index.html [hunk 5]
```diff
@@ -4,16 +4,108 @@
...
+            <button type="button" class="collapsible">View Sets</button>
+            <div class="set-list">
+            </div>
+          </div>
+          <div class="expansion">
+            <img class="series" src="images/sword-shield-logo.png">
```
Comment (shawnkost) [inline index.html]: Needs an `alt` attribute

File: index.html [hunk 6]
```diff
@@ -4,16 +4,108 @@
...
+            <button type="button" class="collapsible">View Sets</button>
+            <div class="set-list">
+            </div>
+          </div>
+          <div class="expansion">
+            <img class="series" src="images/sun-moon-logo.png">
```
Comment (shawnkost) [inline index.html]: Needs an `alt` attribute

File: js/main.js [hunk 7]
```diff
@@ -0,0 +1,146 @@
+const coll = document.querySelectorAll('.collapsible');
```
Comment (shawnkost) [inline js/main.js:1]: Missing a `$` before this!

File: js/main.js [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+      $setList[0].appendChild($setWrapper);
+    }
+    if (cardSets.response.data[i].series === 'Sword & Shield') {
+      const $setWrapper = createSet(i);
+      $setList[1].appendChild($setWrapper);
+    }
+    if (cardSets.response.data[i].series === 'Sun & Moon') {
+      const $setWrapper = createSet(i);
+      $setList[2].appendChild($setWrapper);
+    }
```
Comment (shawnkost) [inline js/main.js]: I would turn the last 2 `if`s into `else if`s. By having these all as `if` statements, you're forcing us to check all 3 conditionals every time, even if only the first one is true. By turning the last 2 into `else if`s, as soon as we find the correct conditional, we won't check the rest

File: js/main.js [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+    data.cardCollection = [];
+    for (let i = 0; i < $setList.length; i++) {
+      $setList[i].style.display = 'none';
+    }
+    $displayCollectionCard.setAttribute('src', 'images/pokemon-card-backside.png');
+  }
+});
+
+function clearCardCollection() {
+  $cardWrapper.innerHTML = '';
```
Comment (shawnkost) [inline js/main.js]: You don't want to use `innerHTML`. It's a security risk. It allows other people to inject scripts and other malicious attacks into your code

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good so far Tam! Some changes requested here

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**